### PR TITLE
[11.0] [MIG] partner_firstname: Migration to 11.0

### PR DESCRIPTION
**Pull request by etobella**
Migration to 11.0

### CODE REVIEW

File: partner_firstname/models/base_config_settings.py [hunk 1]
```diff
@@ -56,9 +58,11 @@ def _onchange_partner_names_order(self):
         self.partner_names_order_changed = self._compute_names_order_changed()
```
Comment (JordiBForgeFlow) [inline partner_firstname/models/base_config_settings.py]: This assignment already occurs in method `_compute_names_order_changed`. As a consequence, it is failing to hide the button "Recalculate names". If you just call to `self._compute_names_order_changed()` it works.

File: partner_firstname/README.rst [hunk 2]
```diff
@@ -47,7 +47,7 @@ supply the *last name* and *first name* (just once per contact).
 
 .. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
    :alt: Try me on Runbot
-   :target: https://runbot.odoo-community.org/runbot/134/10.0
+   :target: https://runbot.odoo-community.org/runbot/134/11.0
```
Comment (JordiBForgeFlow) [inline partner_firstname/README.rst]: Change the readme also, from:

> You can configure some common name patterns for the inverse function
in Settings > Configuration > General settings

To:
> You can configure some common name patterns for the inverse function
in Settings > General settings

**Review by JordiBForgeFlow — CHANGES_REQUESTED** [review]

**Review by rgarnau — COMMENTED** [review]
LGTM :+1:

**Review by rgarnau — APPROVED** [review]

**Review by JordiBForgeFlow — APPROVED** [review]
:+1: Technical and functional review

**Comment by pedrobaeza** [comment]
Please squash together "OCA Transbot..." commits.

**Comment by yajo** [comment]
Runbot is red, could you check that too please?

**Comment by HviorForgeFlow** [comment]
LGTM :+1:

**Review by HviorForgeFlow — APPROVED** [review]

**Comment by etobella** [comment]
@Yajo runbot is green now.

**Review by njeudy — COMMENTED** [review]
- Code check and real tests: Ok for me

**Review by njeudy — APPROVED** [review]
Code check and real tests: Ok for me

**Comment by cliffkujala** [comment]
Not specifically a v11.0 problem, but the README file for partner_firstname links to non HTTPS images so it throws a mixed content warning when viewing the App in the backend.  Problem starts at line 108.

Should be:
```
.. image:: https://odoo-community.org/logo.png
   :alt: Odoo Community Association
   :target: https://odoo-community.org

This module is maintained by the OCA.

OCA, or the Odoo Community Association, is a nonprofit organization whose
mission is to support the collaborative development of Odoo features and
promote its widespread use.

To contribute to this module, please visit https://odoo-community.org.
```

But it is all http:// below line 108.

**Comment by pedrobaeza** [comment]
@cliffkujala Fixed.

**Pull request merged**